packages/blunderbuss/src/blunderbuss.ts

```diff
@@ -54,158 +56,169 @@ export const sleep = (ms: number) => {
 };
 
 export function blunderbuss(app: Application) {
-  app.on(
-    [
-      'issues.opened',
-      'issues.reopened',
-      'issues.labeled',
-      'pull_request.opened',
-      'pull_request.reopened',
-      'pull_request.labeled',
-    ],
-    async context => {
-      const config = (await context.config(
+  const events = [
+    'issues.opened',
+    'issues.reopened',
+    'issues.labeled',
+    'pull_request.opened',
+    'pull_request.reopened',
+    'pull_request.labeled',
+  ];
+  app.on(events, async context => {
+    let config: Configuration = {};
+    try {
+      // Reading the config requires access to code permissions, which are not
+      // always available for private repositories.
+      config = (await context.config<Configuration>(
         CONFIGURATION_FILE_PATH,
         {}
-      )) as Configuration;
-      const issue = (context.payload.issue
-        ? context.payload.issue
-        : context.payload.pull_request) as Issue;
+      ))!;
+    } catch (err) {
+      err.message = `Error reading configuration: ${err.message}`;
+      logger.error(err);
+    }
+    config = config || {};
+
+    const issue: Issue = context.payload.issue || context.payload.pull_request;
+
+    // If this is a PR, and it's in draft mode, don't assign it
+    if (issue.draft === true) {
```

Comment:
Are we sure we want to ignore draft PRs, even when triggered with the issue?

Reply:
I mean, "sure" is a funny word :) This came up in the chat with @lesv and I thought it sounded reasonable.  Are there situations where you think users would wan to assign draft PRs?  I always looked at draft as a way to signal "don't look at me yet, just stashing stuff here"

Reply:
I mean "label" - if a user adds the label while the PR is in draft, should we assign it anyway?

Reply:
Which "label" does that make sense for?  If it's draft, they are saying don't review, or they want a review from someone specific.  They might still want to use kokoro for a bit to test, they can add those labels if they are on the team.  We just don't get bothered until they think they are ready to merge.

Reply:
I could see the confusion there.  You apply the label specifically to get some blunderaction, and it does nothing.  That having been said - I think this is a reasonable restriction.